engine/src/main/resources/org/pentaho/di/job/entries/ping/messages/messages_en_US.properties

```diff
@@ -27,5 +27,5 @@ JobPing.CanNotLoadXML=Unable to load job entry of type 'ping' from XML node
 JobPing.CanNotLoadRep=Unable to load job entry of type 'ping' exists from the repository for id_jobentry={0}
 JobPing.CanNotSaveRep=Unable to save job entry of type 'ping' to the repository for id_job={0}
 JobPing.ClassicPingInterrupted=classic ping process interrupted
-JobPing.NrPackets.Info=Enter numeric value only
+JobPing.NumericInfo=Enter numeric value only
```

Comment:
@AAravind27 do you know who is responsible for generating the messages for the other languages messages_ja_JP.properties, messages_de_DE.properties?

Reply:
Someone answered my question. Basically, it is decided at the end of the release. No action here.